src/pages/iou/IOUModal.js

```diff
@@ -157,6 +159,21 @@ class IOUModal extends Component {
         }
     }
 
+    /**
+     * Decides our animation type based on whether we're increasing or decreasing
+     * our step index.
+     * @returns {String}
+    */
+    getDirection() {
+        if (this.state.previousStepIndex < this.state.currentStepIndex) {
+            return 'in';
+        }
+        if (this.state.previousStepIndex > this.state.currentStepIndex) {
+            return 'out';
+        }
+        return null;
```

Comment:
I don't think we should return null, the Component default is `in` and this would override it. Let's make in the default here too.

Reply:
The last return is only called when `previousStepIndex === currentStepIndex`, we don't want the default `in` since that would mean the first step (once we open the modal) would animate.

I think we can put this in a `(this.state.previousStepIndex === this.state.currentStepIndex)` conditional instead? It's the same logic but should be easier to understand.

Reply:
Ah good point. Yeah, I think that would clear it up a bit. Maybe add a comment too.